src/main/kotlin/racingcar/RegisterCarInRacingFactory.kt

```diff
@@ -0,0 +1,9 @@
+package racingcar
+
+object RegisterCarInRacingFactory {
+    fun registerRacingParticipationCars(registeredNames: List<String>): RacingParticipatingCars {
+        return RacingParticipatingCars(
+            registeredNames.map { name -> Car(name, RacingCarMoveController()) }
+        )
```

Comment:
```suggestion
        val cars = registeredNames.map { name -> Car(name, RacingCarMoveController()) }
        return RacingParticipatingCars(cars)
```
난 이게 더 나은듯? 근데 위에도 나쁘지않아

Reply:
진수씨 map을 아주 잘 쓰시군여..? 나만 repeat forEach 광공이였어.. map이 아주 좋은,, 놈이엿군여 덕분에 배워갑니다